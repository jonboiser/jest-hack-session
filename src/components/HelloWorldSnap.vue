<template>

  <div>
    Hello World {{name}}!
  </div>

</template>


<script>

export default {
  name: 'HelloWorldSnap',
  props: ['name'],
};

</script>
