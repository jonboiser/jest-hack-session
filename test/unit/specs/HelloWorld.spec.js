import HelloWorldSnap from '@/components/HelloWorldSnap';
import { shallow } from '@vue/test-utils';

describe('HelloWorldSnap.vue', () => {
  it('should render correctly with props', () => {
    const wrapper = shallow(HelloWorldSnap, {
      propsData: {
        name: 'Plato',
      },
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
