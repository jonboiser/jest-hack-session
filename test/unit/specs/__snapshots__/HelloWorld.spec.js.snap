// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorldSnap.vue should render correctly with props 1`] = `
<div>
    Hello World Plato!
</div>
`;
